vansh.py

```diff
@@ -0,0 +1 @@
+print("hello everyone")
```

Comment:
bhushan , can you please add some description to this

Reply:
yeah sure i'll do it right away